torch/masked/maskedtensor/_ops_refs.py

```diff
@@ -509,3 +509,22 @@ def _sparse_coo_tensor_with_dims_and_tensors(func, *args, **kwargs):
 def is_same_size(func, *args, **kwargs):
     _check_args_kwargs_length(args, kwargs, f"__torch_dispatch__, {func}", len_args=2)
     return _get_data(args[0]).is_same_size(_get_data(args[1]))
+
+
+@register_dispatch_func([torch.ops.aten._is_any_true])
+def _is_any_true(func, *args, **kwargs):
+    _check_args_kwargs_length(args, kwargs, f"__torch_dispatch__, {func}", len_args=1, len_kwargs=0)
+    data = _get_data(args[0])
+    mask = _maybe_get_mask(args[0])
+    if mask is None:
+        raise ValueError(f"__torch_dispatch__, {func}: expected a mask tensor")
+    if data.dtype != torch.bool:
+        raise ValueError(f"__torch_dispatch__, {func}: expected a boolean tensor")
+    if data.is_sparse:
+        raise ValueError(
+           f"MaskedTensors with sparse data do not have {func}"
+        )
+
+    if func(mask):
+        return MaskedTensor(func(data & mask), torch.tensor(True))
+    return MaskedTensor(torch.tensor(0), torch.tensor(False))
```

Comment:
Could this function just be `return func(data & mask)`

Reply:
Yes that would make more sense.

Reply:
The overload is supposed to return a masked tensor but it is true that if there is no specified true value, then the result is the just False. I'll still wrap the result in a MaskedTensor for coherence, otherwise `MaskedTensor.__torch_function__` will still try to wrap the result in a MaskedTensor and fail.

Reply:
>  otherwise MaskedTensor.__torch_function__ will still try to wrap the result in a MaskedTensor and fail.

Hmm I think its still worth thinking about, as we could simply change the `__torch_function__` to relax the condition.

In theory a masked tensor with all True mask should behave as-if it were a plain tensor, but performance-wise maybe we prefer plain tensors?

Reply:
Nevermind, __torch_function__ does not enforce subclass, I'll fix this today or tomorrow.